app/consent-form/_components/chapter/ChapterIcons.tsx

```diff
@@ -621,17 +628,30 @@ const ConsentDisabled = () => {
 type ChapterIconsProps = {
   icon: Icon
   state: State
+  className?: string
 }
 
-const ChapterIcon = ({ icon, state }: ChapterIconsProps) => {
+const ChapterIcon = ({ icon, state, className }: ChapterIconsProps) => {
   if (icon === 'introduction') {
```

Comment:
Could a switch statement instead of else if's be implemented instead?

Reply:
I tried that first but it didn’t really work since it “switches” over multiple things

Reply:
Okay! Good to know!